integration-tests/api/helpers/call-helpers.js

```diff
@@ -0,0 +1,95 @@
+const { useApi } = require("../../helpers/use-api");
+
+const header = {
+  headers: {
+    authorization: "Bearer test_token",
+  },
+};
+
+const resolveCall = async (path, payload, header) => {
+  const api = useApi();
+  let res;
+  try {
+    res = (await api.post(path, payload, header)).status;
+  } catch (e) {
+    res = e.response.status;
+  }
+  return res;
+};
+
+const determineFail = (actual, expected, path) => {
```

Comment:
Used to both fail test if required, but also print the path for which we failed. Using console.log here to avoid it referring to where in the source code the error was raised (console.error)